setup.py

```diff
@@ -65,8 +65,8 @@
     'Topic :: Software Development'
   ],
   packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
-  install_requires=["six", "securesystemslib==0.10.8", "attrs",
-                    "python-dateutil", "iso8601"],
+  install_requires=["six", "cryptography>=2.1.3", "securesystemslib==0.10.8",
```

Comment:
You explain in the pull request summary that the cryptography dependency is set to `>=2.1.3` because that's the minimum version of cryptography supported by `securesystemslib`.

Should you add a note/comment in `setup.py` to remind of that fact?    Suppose `securesystemslib` increases the minimum version of cryptography, will in-toto have to follow suit?

Reply:
I'll add a comment.